dbbackup/dbcommands.py

```diff
@@ -282,12 +283,12 @@ def run_command(self, command, stdin=None, stdout=None):
 
     def read_file(self, filepath, stdout):
         """ Read the specified file to stdout. """
-        print("  Reading: %s" % filepath)
+        sys.stdout.write("  Reading: %s" % filepath)
```

Comment:
Ah okay, this isn't part of BaseCommand, so it needs to do something for newlines for Python2... it's only Python3 that automatically adds newlines.